File: trail2/색종이의 총 넓이/the-total-area-of-colored-paper.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        // Please write your code here.
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer st;

        int n = Integer.parseInt(br.readLine());

        final int OFFSET = 100;
        final int MAX_W = 201;

        int[][] grid = new int[MAX_W + 1][MAX_W + 1];

        for (int i = 0; i < n; i += 1) {
            st = new StringTokenizer(br.readLine());

            int x = Integer.parseInt(st.nextToken()) + OFFSET,
                y = Integer.parseInt(st.nextToken()) + OFFSET;

            for (int j = x; j < x + 8; j += 1) {
                for (int k = y; k < y + 8; k += 1) {
                    grid[j][k] = 1;
                }
            }
        }

        int cnt = 0;

        for (int i = 0; i < MAX_W; i += 1) {
            for (int j = 0; j < MAX_W; j += 1) {
                if (grid[i][j] > 0) cnt += 1;
            }
        }

        bw.write(String.valueOf(cnt));
        bw.flush();
        bw.close();



    }
}

File: trail2/색종이의 총 넓이/the-total-area-of-colored-paper.py
n = int(input())
points = [tuple(map(int, input().split())) for _ in range(n)]
x, y = zip(*points)
x, y = list(x), list(y)

# Please write your code here.
SIZE = 8
OFFSET = 100
grid = [[0 for _ in range(OFFSET * 2 + 1)] for _ in range(OFFSET * 2 + 1)]

for i in range(n):
    for row in range(x[i], x[i] + SIZE):
        for col in range(y[i], y[i] + SIZE):
            grid[row + OFFSET][col + OFFSET] = 1

res = sum(grid, [])
print(res.count(1))